src/calendar/styles.scss

```diff
@@ -83,14 +90,12 @@
       cursor: pointer;
       color: calendar.$grid-nonmonth-day-color;
       &::after {
-        border-start-start-radius: awsui.$border-radius-item;
-        border-start-end-radius: awsui.$border-radius-item;
-        border-end-start-radius: awsui.$border-radius-item;
-        border-end-end-radius: awsui.$border-radius-item;
+        @include item-border-radius;
       }
       &.calendar-date-current-page {
         color: calendar.$grid-date-color;
         &:hover {
+          @include item-border-radius;
```

Comment:
This is the rule that was missing, the rest were refactored just to avoid repetitions.